instrumentation/opentelemetry-instrumentation-mysqlclient/tests/test_mysqlclient_integration.py

```diff
@@ -96,6 +96,42 @@ def test_instrument_connection(self, mock_connect):
         spans_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans_list), 1)
 
+    @mock.patch("opentelemetry.instrumentation.dbapi.instrument_connection")
+    @mock.patch("MySQLdb.connect")
+    # pylint: disable=unused-argument
+    def test_instrument_connection_enable_commenter(
+        self,
+        mock_connect,
+        mock_instrument_connection,
+    ):
+        cnx = MySQLdb.connect(database="test")
+        cnx = MySQLClientInstrumentor().instrument_connection(
+            cnx,
+            enable_commenter=True,
+            commenter_options={"foo": True},
+        )
+        cursor = cnx.cursor()
+        cursor.execute("SELECT * FROM test")
+        kwargs = mock_instrument_connection.call_args[1]
+        self.assertEqual(kwargs["enable_commenter"], True)
+        self.assertEqual(kwargs["commenter_options"], {"foo": True})
+
+    @mock.patch("opentelemetry.instrumentation.dbapi.wrap_connect")
+    @mock.patch("MySQLdb.connect")
+    # pylint: disable=unused-argument
+    def test__instrument_enable_commenter(
```

Comment:
```suggestion
    def test_instrument_enable_commenter(
```

was this intentional?

Reply:
It tests the method `_instrument` so I did do this intentionally with the double underscore.

Reply:
I think I personally prefer single underscore since I mentally parse the leading underscore as "protected". To avoid bikeshed, I'm OK with whatever as long as it's consistent with the style of the project. I think a few tests in here were already `test_instrument_xx`, not sure what other packages in this repo do

Reply:
We can do singles 👍

Reply:
Single unders in [e0ecef0](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2941/commits/e0ecef01b9babd5e9e6ae6c07d09c92360b33e5d)